torch_uncertainty/models/wideresnet/std.py

```diff
@@ -186,7 +198,10 @@ def wideresnet28x10(
     in_channels: int,
     num_classes: int,
     groups: int = 1,
+    dropout_rate: float = 0.0,
```

Comment:
I'd leave 0.3 as the default value since WideResNet is intended to be trained with dropout if I recall the paper correctly.